app/containers/zazu.js

```diff
@@ -80,7 +73,17 @@ const Zazu = React.createClass({
           />
       </div>
     )
-  },
-})
+  }
+}
+
+Zazu.propTypes = {
```

Comment:
Yes it works like that https://facebook.github.io/react/docs/typechecking-with-proptypes.html